### DISCUSSION TITLE
hexo g error报错 无法加载css help！！！

### DISCUSSION THREAD

**Discussion by gonisan** [hexojs/hexo#4969] [Question / Answer] [answered]
![20220514135714](https://user-images.githubusercontent.com/62231811/168413184-b9bc0931-4099-4cde-a36b-336c6ca4017a.png)

第一次使用hexo搭建博客，在hexo g时遇见了这个问题

**Comment by dingliusheng** [comment]
me too

**Comment by MHuiG** [comment, marked as answer]
https://github.com/hexojs/hexo/issues/4968
